src/gui/MainWindow.cpp

```diff
@@ -1102,6 +1101,14 @@ void MainWindow::updateViewMenu()
 			toInt() ? true : false );
 	m_viewMenu->addAction(qa);
 	*/
+
+	qa = new QAction(tr( "Enable note labels in piano roll" ), this);
+	qa->setData("printnotelabels");
+	qa->setCheckable( true );
+	qa->setChecked( ConfigManager::inst()->value( "ui", "printnotelabels" ).
+			toInt() ? true : false );
```

Comment:
I would personaly have left out the ````? true : false ```` as i feel it makes the code harder to read.

After further inspection you have followed the style of the surounding code, That is far more important :)